src/all/keyboardEvents.js

```diff
@@ -0,0 +1,4 @@
+import * as KEYMAP from '../keyConstants';
```

Comment:
Just one more thing, I think we don't need this file, we could just have `src/keyboardEvents.js` export `KEYMAP` and make package.json `main` and `jsnext:main` point to that instead. We only create a separate files like this when there are multiple files exporting values, without a main file that already imports them all. Examples are `metal` and `metal-dom`. In this case `src/keyboardEvents.js` is already the main file, and it already imports `src/keyConstants`, so it should be enough.